spec/quality_spec_es.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+if defined?(Encoding) && Encoding.default_external.name != "UTF-8"
+  # Poor man's ruby -E UTF-8, since it works on 1.8.7
+  Encoding.default_external = Encoding.find("UTF-8")
+end
+
+RSpec.describe "La biblioteca si misma" do
+
+  def check_for_expendable_words(filename)
+    failing_line_message = []
+    useless_words = %w[
+      actualmente
+      básicamente
+      claramente
+      sólo
+      solamente
+      obvio
+      obviamente
+      fácil
+      fácilmente
+      sencillamente
+      simplemente
+    ]
+    pattern = /\b#{Regexp.union(useless_words)}\b/i
+
+    File.readlines(filename).each_with_index do |line, number|
+      next unless word_found = pattern.match(line)
+      failing_line_message << "#{filename}:#{number.succ} tiene '#{word_found}'. No use estos tipos de modificantes."
+    end
+
+    failing_line_message unless failing_line_message.empty?
```

Comment:
needs to always return an array for the be_well_formed matcher

Reply:
@segiddins I'm not sure if I follow-- is `error_messages` ever not an array?

Reply:
if `failing_line_message` is empty, the method will return `nil`, and the `be_well_formed` matcher will try to call `nil.empty?` which will raise an exception

Reply:
@segiddins So, I commented out the `unless` clause on lines 32 and 44: `failing_line_message #unless failing_line_message.empty?`. 

I thought if I'm always returning `failing_line_message` in the `check_for_expendable_words` and `check_for_specific_pronouns` methods, raising the exception would be avoided since I'd always be returning an array...? 
But the specs break when I comment out that `unless` clause.🤔

Reply:
ah my bad, I was misunderstanding how the method was being used :/ Ignore me